src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -70,6 +73,13 @@ def sql_alter_from(self, catalog):
         )
 
 
+def parse_table_props(tbl_props: string) -> {}:
```

Comment:
don't let this public function dangling on a module level, it has to be a private method in the `TablesCrawler`. also add a unit test, so that we know when parsing of table properties is broken.